edx_dl/parsing.py

```diff
@@ -239,21 +240,11 @@ def extract_courses_from_html(self, page, BASE_URL):
         Extracts courses (Course) from the html page
         """
         soup = BeautifulSoup(page)
-
-        # First, try with new course structure (as of December 2017).  If
-        # that doesn't work, we fallback to an older course structure
-        # (released with version 0.1.6). If even that doesn't work, then we
-        # try with the oldest course structure (that was current before
-        # version 0.1.6).
-        #
-        # rbrito---This code is ugly.
```

Comment:
I don't get why you deleted this comment of mine. This is a comment that documents why something is being done in the code that isn't said in the code... Do you have a reason to remove that?

Reply:
Even I myself don't get why. It seems to be a mistake in the middle of trying things. Sorry for the mistake.

Reply:
By the way, after this pull request was made, I added --english-only option.

Reply:
I would strongly suggest that you refrain from adding any new command-line options. There should be a VERY strong reason affecting most of the users, not just you, to do so. Every option complicates things further and further and it's repo owners (not one-time contributors) who will have to support that complexity later on. Please bear that in mind.